spec/integration_spec.rb

```diff
@@ -9,17 +9,17 @@
     after{ expect(controller.bird).to eq(mockingbird) }
 
     it "finds model by id" do
-      get :show, params: { id: "mockingbird" }
+      get :show, { id: "mockingbird" }
     end
 
     it "finds model by bird_id" do
-      get :show, params: { bird_id: "mockingbird" }
+      get :show, { bird_id: "mockingbird" }
     end
   end
 
   it "builds bird if id is not provided" do
     bird = double("Bird")
-    expect(Bird).to receive(:new).and_return(bird)
+    expect(Bird).to receive(:new).with({}).and_return(bird)
```

Comment:
we are making this expectation more specific here with the `with`. It was failing without the hash because `new` was not expecting arguments